backend/app/main.py

```diff
@@ -53,3 +60,47 @@ async def db_is_alive(session: Session = Depends(get_session)):
         raise HTTPException(
             status_code=status.HTTP_503_SERVICE_UNAVAILABLE, detail="DB is unreachable"
         )
+
+
+@app.post(
+    "/create_document",
+    response_model=DocumentPublic,
+    status_code=status.HTTP_201_CREATED,
+)
+async def create_document(session: Session = Depends(get_session)):
+    # To be created in the database
+    results = session.execute(text("SELECT create_document();"))
+    document_id = results.one()[0]  # should be only one row, one column of results
```

Comment:
@raphaellaude I tried to return the whole document object in the one udf, but it was coming back with unlabeled columns so accessing it as doc.document_id was failing. Gonna keep futzing with it bc imo silly to do another trip to the database for this, but I think this is ok to merge as-is.